src/App.css

```diff
@@ -10,4 +10,4 @@
   justify-content: center;
   flex-direction: column;
   gap: 50px;
-}
```

Comment:
Please remove unused pieces of code from your codebase in order to make your codebase clean